external/radiation/radiation/getdata.py

```diff
@@ -292,6 +293,20 @@ def sw(lookup_dir: str):
     return sw_dict
 
 
+def sigma(restart_dir, p_ref=101325.0):
```

Comment:
Rename to highlight I/O?

```suggestion
def load_hybrid_coordinate(restart_dir, p_ref=101325.0):
```

Reply:
Hmm, my thinking was that the functions in `getdata` would be called in practice by `getdata.<func>` which accomplishes what you're suggesting. Alternatively we could rename all of the functions in the module and call them without the module name.

Reply:
That is a good naming approach.